client/state/posts/actions.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Internal dependencies
+ */
+import wpcom from 'lib/wp';
+import {
+	POST_RECEIVE,
+	POSTS_RECEIVE,
+	POSTS_REQUEST,
+	POSTS_REQUEST_SUCCESS,
+	POSTS_REQUEST_FAILURE
+} from 'state/action-types';
+
+/**
+ * Returns an action object to be used in signalling that a post object has
+ * been received.
+ *
+ * @param  {Object} post Post received
+ * @return {Object}      Action object
+ */
+export function receivePost( post ) {
+	return {
+		type: POST_RECEIVE,
+		post
```

Comment:
>It seems like we could prevent some duplication of code and error if we simply called the multi-post action `POSTS_RECEIVE` here, passing in `[ post ]` instead of `post`

I like this idea. I think it'd help significantly in the reducer functions to avoid accounting for a variety of inputs, when in fact its the difference of being wrapped as an Array. We could still expose the action creator for receiving a single post, so the developer doesn't need to concern themselves with the underlying action.

I'll plan to take a stab at this approach later this morning.